pallets/schemas/src/lib.rs

```diff
@@ -221,21 +283,68 @@ pub mod pallet {
 	pub(super) type SchemaNameToIds<T: Config> = StorageDoubleMap<
 		_,
 		Blake2_128Concat,
-		SchemaNamespace,
+		SchemaProtocolName,
 		Blake2_128Concat,
 		SchemaDescriptor,
 		SchemaVersionId,
 		ValueQuery,
 	>;
 
+	/// Storage for mapping names to IDs
```

Comment:
WHy didn't we remove SchemaNameToIds?

Reply:
I think we have usecases that allows fetching schemas and intents by their name but I'm not sure why we want. remove that?